src/components/ProposalListTable.tsx

```diff
@@ -25,12 +26,26 @@ const ProposalListTableRow = ({
     navigate(`/proposals/${proposal.id}`);
   };
 
+  const getProposalCellContents = (shortCode: string) => {
+    let proposalValue;
+
+    switch (shortCode) {
```

Comment:
> Note that `getProposalCellContents()` is implemented with a switch statement because I predict we’ll do something similar for other columns very soon. We may not stick with this syntax but it’s a good bet.

What other columns are you thinking about?

Reply:
@jasonowens I'm specifically thinking this may be the entry point for doing formatting or linkifying URLs.

Reply:
Aha! I had been thinking of introducing a separate component - `<FormattedValue>` or so - but this is interesting. Let me think about it!

Reply:
I think for formatting values we'll want to make a decision on a not-yet-passed attribute of a base field, rather than on the short code.

What I thought you might be getting at is that there are other sets of columns that should be collapsed like this, in which case I might advocate for a more general approach, but we brainstormed for a bit the other day as we were discussing variants and couldn't come up with any other such sets.

So yes, would you mind switching (ha) to an `if` statement, please? Thank you! :blue_heart:

Reply:
> I think for formatting values we'll want to make a decision on a not-yet-passed attribute of a base field, rather than on the short code.

Ah yes, good point.

> What I thought you might be getting at is that there are other sets of columns that should be collapsed like this, in which case I might advocate for a more general approach, but we brainstormed for a bit the other day as we were discussing variants and couldn't come up with any other such sets.

We do have one more (which I should have mentioned above): Proposal Name. The table is simplistically only looking at `proposal_name`, but [on the detail page we cascade to `proposal_title`](https://github.com/PhilanthropyDataCommons/data-viewer/blob/475c87d5c0ef94ee19f2628d013ea885d299a6f3/src/pages/ProposalDetail.tsx#L122-L123). In fact… we should fix that both here on the table and in the [sidebar grid](https://github.com/PhilanthropyDataCommons/data-viewer/blob/475c87d5c0ef94ee19f2628d013ea885d299a6f3/src/components/ProposalListGridItem.tsx#L38).

…which is to say, can we keep this a `switch` after all and I either fix the proposal title as well, or add that as a separate followup task?

Reply:
Excellent point! Okay yes, let's keep the `switch`. Thanks for talking and thinking through this with me!

Once we have a resolution to https://github.com/PhilanthropyDataCommons/service/issues/284 we can follow up; let's not block this PR on that question.

Reply:
@jasonaowen Okay based on our conversation in Zulip just now, it seems like there is an unresolved question as to whether `project_name` and `proposal_project_title` are conceptually related enough that we should do the "org name"-style cascading among them. We're going to discuss it in tomorrow's meeting.

If they aren't related enough, we'll need to remove the Proposal Title cascade from the detail page. If they are related enough, we'll want to adopt the cascade on the two missing places (table and sidebar).

Since in either case that's out of scope of this PR, I'm going to reduce it to an `if` and we can switch (lol you're right that is funny) to a `switch` if we decide that path.

Reply:
@jasonaowen Ah I didn't see your reply as I was drafting mine… okay yes, it's easiest just to leave it in (and I sort of suspect we're going to have multiple proposal name field candidates). Will keep as-is, and thank _you_!